.config-schema.json

```diff
@@ -464,38 +464,50 @@
                 }
             }
         },
+        "checkCache": {
+            "type": "object",
+            "properties": {
+                "limit": {
+                    "description": "the size limit (in items) of the cache for Check (queries and iterators)",
+                    "type": "integer",
+                    "default": "10000",
```

Comment:
@elbuo8 the default here is 10k, but the default for `checkIteratorCache.maxResults` is also 10k - this seems to imply that one DB iterator will use up the entire cache?

Reply:
A cache entry value for an iterator is a slice. `checkIteratorCache.maxResults` limits the size/length of those slices. If a datastore query produces >= `checkIteratorCache.maxResults` we won't cache it. This `limit` is the maximum number of entries in the in memory cache instance.